lib/slug.rb

```diff
@@ -13,11 +14,12 @@ def self.for(string, default = 'topic')
       end
     # Reject slugs that only contain numbers, because they would be indistinguishable from id's.
     slug = (slug =~ /[^\d]/ ? slug : '')
+    slug = slug.length >= MAX_SLUG_LENGTH ? '' : slug
```

Comment:
are we sure about this, why not truncate?

Reply:
I was definitely out of my mind...will fix